### DISCUSSION TITLE
I would like to contribute FL & distributed training, how to get started?

### DISCUSSION THREAD

**Discussion by chaoyanghe** [openmlsys/openmlsys-zh#172] [Q&A] [answered]
Hi OpenMLSys Team,

This is a nice effort. I am also writing a book for federated and distributed training, which is also my PhD thesis. It would be great if I can contribute part of it into your book.

Here is my background:

CTO & Co-founder of FedML.ai (https://fedml.ai)
PhD in Computer Science from University of Southern California
Homepage: https://chaoyanghe.com

**Comment by eedalong** [comment, marked as answer]
Ha! I didnt notice here is a discussion opened, my bad. Happy to hear that you are interested, we will reach out to you very soon!

**Comment by luomai** [comment]
@chaoyanghe It is great to hear from you. We are also very excited about your interest in joining our effort for making this book happen. Could you connect me with my [Wechat account](https://github.com/openmlsys/openmlsys-zh/blob/main/info/editors.md)? I would be keen on starting our discussion from there.
